lib/uplinkc/apikey.go

```diff
@@ -27,7 +27,7 @@ func parse_api_key(val *C.char, cerr **C.char) (apikeyHandle C.APIKeyRef_t) {
 func serialize_api_key(apikeyHandle C.APIKeyRef_t) *C.char {
 	apikey, ok := universe.Get(apikeyHandle._handle).(libuplink.APIKey)
 	if !ok {
-		return C.CString("")
+		return C.CString("invalid apikey")
```

Comment:
This should be returned as an error not as the result.

Reply:
:+1: